test/built-ins/RegExp/named-groups/string-replace-nocaptures.js

```diff
@@ -24,6 +24,7 @@ for (let flags of ["", "u"]) {
   assert.sameValue("bacd", "abcd".replace(re, "$2$1"));
   assert.sameValue("cd", "abcd".replace(re, "$3"));
   assert.sameValue("$<sndcd", "abcd".replace(re, "$<snd"));
+  assert.sameValue("$<sndacd", "abcd".replace(re, "$<snd$1"));
```

Comment:
isn't this a case for `test/built-ins/RegExp/named-groups/string-replace-unclosed.js`? It seems like an unclosed name capture.

The test is not incorrect, thou. I wonder if it becomes hard to find it later if anything changes or navigating a test failure.

Reply:
It's not really unclosed, since it was never opened :) In particular, the RegExp doesn't have any captures. That's why it's in the -nocaptures file.

Reply:
About finding the test: I don't really know how you do it, but I have trouble finding tests by name; the directory is useful, but beyond that, I just find myself grepping around. I guess that informs how I divide up tests into files.